src/test/munit/tcp-ssl-test-suite.xml

```diff
@@ -13,24 +13,25 @@ http://www.mulesoft.org/schema/mule/tls http://www.mulesoft.org/schema/mule/tls/
 http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd">
 
     <munit:config name="tcp-ssl-test-suite.xml" minMuleVersion="4.3.0"/>
+    <configuration-properties file="configs.properties" />
 
     <munit:dynamic-port propertyName="port"/>
     <munit:dynamic-port propertyName="port2"/>
 
-    <tls:context name="globalTlsContext">
-        <tls:trust-store path="tls/ssltest-cacerts.jks" password="changeit"/>
-        <tls:key-store path="tls/ssltest-keystore.jks" keyPassword="changeit" password="changeit"/>
+    <tls:context name="globalTlsContext" enabledProtocols="TLSv1.2">
```

Comment:
why is it necessary to set enabledProtocols?

Reply:
@sabecasissf I added it since it is the minimum FIPS-compliant version.